### PR TITLE
[PERF-318] Run Pipeclean Test on github Actions

### PR DESCRIPTION
**Pull request by JBrenesSimpat**
Pipeclean Test on VM

### CODE REVIEW

File: .github/workflows/Pipeclean_test.yml [hunk 1]
```diff
@@ -0,0 +1,36 @@
+# SPDX-License-Identifier: Apache-2.0
+# Licensed to the Ed-Fi Alliance under one or more agreements.
+# The Ed-Fi Alliance licenses this file to you under the Apache License, Version 2.0.
+# See the LICENSE and NOTICES files in the project root for more information.
+
+name: Run Pipeclean Test
```
Comment (semalaiappan) [inline .github/workflows/Pipeclean_test.yml:6]: Remove Underscore in file name

File: .github/workflows/Pipeclean_test.yml [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+on:
+  workflow_dispatch:
+
+env:
+    AzureTestVmUsername: ${{ secrets.AZURE_TEST_VM_USER_NAME }}
```
Comment (semalaiappan) [inline .github/workflows/Pipeclean_test.yml:11]: use uppercase with underscore for env file name  AzureTestVmUsername

File: .github/workflows/Pipeclean_test.yml [hunk 3]
```diff
@@ -0,0 +1,36 @@
...
+    steps:
+      - name: Checkout code
+        uses: actions/checkout@8ade135a41bc03ea155e62e844d188df1ea18608  # v4.1.0
+
+      - name: Execute Test Runner
+        if: ${{ env.AzureTestVmUsername != '' && env.AzureTestVmPassword != '' }}
+        shell: powershell
+        working-directory: ./eng/
+        run: |
+
```
Comment (semalaiappan) [inline .github/workflows/Pipeclean_test.yml:27]: Remove whitespace lines

**Review by semalaiappan — CHANGES_REQUESTED** [review]
Review comments updated

File: .github/workflows/Pipeclean_test.yml (hunk 2 above)
Comment (JBrenesSimpat) [reply .github/workflows/Pipeclean_test.yml:11]: these env variable names are used in this way by the AzureTestLab.ps1 file that is also used in TC, that is the reason why this name was used

**Review by semalaiappan — APPROVED** [review]

**Pull request merged**